### PR TITLE
Fix/ubuntu build

### LINKED ISSUE #32: Build fails on Ubuntu 17.04
In the current project the build fails when the command `make` is executed in the root of the project.
The issue is that Ubuntu does not come with versions of included libs that are compiled with `-fPIC` which is required on Ubuntu.

### PR DESCRIPTION
**Pull request by fabulousduck**
This fixes #32 for ubuntu.

check if this works for your system aswell as i may need to make this ubuntu specific if this breaks it on your end

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,20 +1,25 @@
 # TODO(#25): Generate header dependencies for Makefile
 #
 # See `nasm -M` for more info on that
+CC=gcc
+cc=nasm
```
Comment (rexim) [inline Makefile]: Can we call it `ASM` instead of `cc`?

**Review by rexim — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (rexim) [reply Makefile]: Or something else that is conventional for assembler?

Comment (rexim) [reply Makefile]: I guess it should be `AS`

File: Makefile [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
 OBJ_FILES=$(ASM_FILES:.asm=.o)
 
 TEST_ASM_FILES=$(wildcard test/*.asm)
 TEST_OBJ_FILES=$(TEST_ASM_FILES:.asm=.o)
 
+
+
 src/webapp: $(OBJ_FILES)
-	gcc $(OBJ_FILES) -o src/webapp
+	$(CC) -no-pie $(OBJ_FILES) -o src/webapp
```
Comment (rexim) [inline Makefile:16]: Should be probably double dash like `--no-pie`

**Comment by fabulousduck** [comment]
closing this in favour of pr made by @richardbraun.

**Pull request closed without merging**